Fix wrong assertion in MetricsKernelExtensionFactoryIT

davidegrohmann opened the pull request.

davidegrohmann committed:
Fix wrong assertion in MetricsKernelExtensionFactoryIT

spacecowboy commented on enterprise/metrics/src/test/java/org/neo4j/metrics/MetricsKernelExtensionFactoryIT.java:
```diff
@@ -62,6 +64,8 @@
     private static final int METRICS_VALUE = 1;
 
     @Rule
+    public final RepeatRule repeatRule = new RepeatRule();
```
Left over from testing (as discussed)

davidegrohmann committed:
Improve event based metrics test

Ensure we wait for the event to be written in the csv file before
shutting down the cluster.
